datalake/src/test/java/com/sequenceiq/datalake/service/sdx/SdxEventsServiceTests.java

```diff
@@ -0,0 +1,213 @@
+
+        List<CDPStructuredEvent> result = sdxEventsService.getDatalakeAuditEvents(RESOURCE_CRN,
+                Collections.singletonList(StructuredEventType.NOTIFICATION), TEST_PAGE, TEST_SIZE);
+
+        assertNotNull(result);
+        assertEquals(1, result.size());
+        assertTrue(result.get(0).getOperation().getEventType().equals(StructuredEventType.NOTIFICATION));
+    }
+
+    @Test
+    void testGetAuditEventsWithCBAndDlEvents() {
+        Page<CDPStructuredEvent> mockPage = mock(Page.class);
+
+        when(mockPage.getContent()).thenReturn(List.of(createTestCDPStructuredEvent(StructuredEventType.NOTIFICATION)));
+        when(mockCdpStructuredEventDBService.getPagedEventsOfResources(eq(List.of(StructuredEventType.NOTIFICATION)), any(), any()))
+                .thenReturn(mockPage);
+        when(eventV4Endpoint.getCloudbreakEventListByStack(any(), any(), any(), any()))
+                .thenReturn(List.of(createCloudbreakEventV4Response()));
+
+        List<CDPStructuredEvent> result = sdxEventsService.getDatalakeAuditEvents(RESOURCE_CRN,
+                Collections.singletonList(StructuredEventType.NOTIFICATION), TEST_PAGE, TEST_SIZE);
+
+        assertNotNull(result);
+        assertEquals(2, result.size());
+        assertEquals(2, result.stream().filter(notification -> notification.getOperation().getEventType().equals(StructuredEventType.NOTIFICATION)).count());
+    }
+
+    @Test
+    void testGetAuditEventsWithCBAndDlEventsSort() {
+        Page<CDPStructuredEvent> mockPage = mock(Page.class);
+
+        when(mockPage.getContent()).thenReturn(List.of(createTestCDPStructuredEvent(StructuredEventType.NOTIFICATION, 5L),
+                createTestCDPStructuredEvent(StructuredEventType.NOTIFICATION, 2L)));
+        when(mockCdpStructuredEventDBService.getPagedEventsOfResources(eq(List.of(StructuredEventType.NOTIFICATION)), any(), any()))
+                .thenReturn(mockPage);
+        when(eventV4Endpoint.getCloudbreakEventListByStack(any(), any(), any(), any()))
+                .thenReturn(List.of(createCloudbreakEventV4Response(1L)));
+
+        List<CDPStructuredEvent> result = sdxEventsService.getDatalakeAuditEvents(RESOURCE_CRN,
+                Collections.singletonList(StructuredEventType.NOTIFICATION), TEST_PAGE, TEST_SIZE);
+
+        assertNotNull(result);
+        assertEquals(3, result.size());
+        assertEquals(3, result.stream().filter(notification -> notification.getOperation().getEventType().equals(StructuredEventType.NOTIFICATION)).count());
+        assertTrue(result.get(0).getOperation().getTimestamp() < result.get(1).getOperation().getTimestamp());
+        assertTrue(result.get(1).getOperation().getTimestamp() < result.get(2).getOperation().getTimestamp());
+    }
+
+    private CloudbreakEventV4Response createCloudbreakEventV4Response() {
+        return createCloudbreakEventV4Response(1L);
+    }
+
+    private CloudbreakEventV4Response createCloudbreakEventV4Response(Long timestamp) {
```

Comment:
`Long timestamp` is always 1

Reply:
same: 
![image](https://user-images.githubusercontent.com/19725708/148399941-34524a94-d2fb-4ef4-94ac-8b86c5724dfd.png)